src/react-components/avatar-selector.js

```diff
@@ -0,0 +1,119 @@
+    // so we need to force it here.
+    const currRot = this.animation.parentNode.getAttribute('rotation');
+    this.animation.setAttribute('from', `${currRot.x} ${currRot.y} ${currRot.z}`);
+    this.animation.stop();
+    this.animation.handleMixinUpdate();
+    this.animation.start();
+  }
+
+  render () {
+    const avatarAssets = this.props.avatars.map(avatar => (
+      <a-progressive-asset
+        id={avatar.id}
+        key={avatar.id}
+        response-type="arraybuffer"
+        high-src={`./src/assets/avatars/${avatar.models.high}`}
+        low-src={`./src/assets/avatars/${avatar.models.low}`}
+      ></a-progressive-asset>
+    ));
+
+    const avatarEntities = this.props.avatars.map((avatar, i) => (
+      <a-entity key={avatar.id} position="0 0 0" rotation={`0 ${360 * -i / this.props.avatars.length} 0`}>
+        <a-gltf-entity position="0 0 5" rotation="0 0 0" src={'#' + avatar.id} inflate="true">
+          <template data-selector=".RootScene">
+            <a-entity animation-mixer></a-entity>
+          </template>
+          <a-animation
+            attribute="rotation"
+            dur="2000"
+            to={`0 ${this.getAvatarIndex() === i ? 360 : 0} 0`}
+            repeat="indefinite">
+          </a-animation>
+        </a-gltf-entity>
+      </a-entity>
+    ));
+
+    return (
+      <div className="avatar-selector">
+      <span className="avatar-selector__loading">
+        <FormattedMessage id="profile.avatar-selector.loading"/>
+      </span>
+      <a-scene vr-mode-ui="enabled: false" ref={sce => this.scene = sce}>
+        <a-assets>
+          {avatarAssets}
+          <a-asset-item
+            id="meeting-space1-mesh"
+            response-type="arraybuffer"
+            src="./src/assets/environments/MeetingSpace1_mesh.glb"
+          ></a-asset-item>
+        </a-assets>
+
+        <a-entity data-avatar={this.props.avatar}>
+          <a-animation
+            ref={anm => this.animation = anm}
+            attribute="rotation"
+            dur="1000"
+            easing="ease-out"
+            to={`0 ${360 * this.getAvatarIndex() / this.props.avatars.length + 180} 0`}>
+          </a-animation>
+        {avatarEntities}
+        </a-entity>
+
+        <a-entity position="0 1.5 -5.6" rotation="-10 180 0" camera></a-entity>
+
+        <a-entity
+          hide-when-quality="low"
+          light="type: directional; color: #F9FFCE; intensity: 0.6"
+          position="0 5 -15"
+        ></a-entity>
+        <a-entity
+          hide-when-quality="low"
+          light="type: ambient; color: #FFF"
+        ></a-entity>
+        <a-gltf-entity
+          id="meeting-space"
+          src="#meeting-space1-mesh"
+          position="0 0 0"
+        ></a-gltf-entity>
+      </a-scene>
+      <button className="avatar-selector__prev-button" onClick={this.prevAvatar}>
+        <i className="avatar-selector__button-icon material-icons">keyboard_arrow_left</i>
```

Comment:
on my branch i'm ripping out the dependency on material design lite -- we can keep it if you think it's important but if this is the only coupling left probably worth just extracting these icons. (I'm not going to use it for the home landing page)

Reply:
Have you actually been extracting the icons out of MDL? How about using Font Awesome?

Reply:
I've been drawing them myself! :) (except for the little person icon, which I grabbed from google's material design site iirc)

Reply:
Hmm, okay. I'd rather just use Font Awesome if you want to get rid of MDL. But I guess we're going to throw all of this design away and use bespoke assets before release?

Reply:
I too am a bit confused by this. If we plan to replace them we definitely should just be pulling somehting off the shelf. And even if we don't not sure icon design is the best use of any of our time lol

Reply:
sure we can use Font Awesome. i didn't see a mic/speaker (or more recently, a webvr) icon that suited my needs in the public google design icon set so I just drew them. for these angle brackets we can def just use font awesome. in either case i think we want to pack all the necessary assets into the client repo and not link externally.

my main point was dropping the material design lite dependency (which I think is a good idea since we're not using it) will break these icons, so we should extract them, etc. i don't particularly care where they come from or if we make them ourselves etc.